tests/test_compute_funs.py

```diff
@@ -299,94 +308,94 @@ def test_magnetic_field_derivatives(DummyStellarator):
     )
     np.testing.assert_allclose(
-        data["B^zeta_t"][2:-2],
-        B_sup_zeta_t[2:-2],
+        data["B^zeta_t"][4:-4],
+        B_sup_zeta_t[4:-4],
         rtol=rtol,
         atol=atol * np.mean(np.abs(data["B^zeta_t"])),
     )
     np.testing.assert_allclose(
-        data["B^zeta_tt"][2:-2],
-        B_sup_zeta_tt[2:-2],
+        data["B^zeta_tt"][4:-4],
+        B_sup_zeta_tt[4:-4],
         rtol=rtol,
         atol=atol * np.mean(np.abs(data["B^zeta_tt"])),
     )
     np.testing.assert_allclose(
-        data["B_rho_t"][2:-2],
-        B_sub_rho_t[2:-2],
+        data["B_rho_t"][4:-4],
+        B_sub_rho_t[4:-4],
         rtol=rtol,
         atol=atol * np.mean(np.abs(data["B_rho_t"])),
     )
     np.testing.assert_allclose(
-        data["B_rho_tt"][2:-2],
-        B_sub_rho_tt[2:-2],
+        data["B_rho_tt"][4:-4],
+        B_sub_rho_tt[4:-4],
         rtol=rtol,
         atol=atol * np.mean(np.abs(data["B_rho_tt"])),
     )
     np.testing.assert_allclose(
-        data["B_theta_t"][2:-2],
-        B_sub_theta_t[2:-2],
+        data["B_theta_t"][4:-4],
+        B_sub_theta_t[4:-4],
         rtol=rtol,
         atol=atol * np.mean(np.abs(data["B_theta_t"])),
     )
     np.testing.assert_allclose(
-        data["B_theta_tt"][2:-2],
-        B_sub_theta_tt[2:-2],
+        data["B_theta_tt"][4:-4],
+        B_sub_theta_tt[4:-4],
         rtol=rtol,
         atol=atol * np.mean(np.abs(data["B_theta_tt"])),
     )
     np.testing.assert_allclose(
-        data["B_zeta_t"][2:-2],
-        B_sub_zeta_t[2:-2],
+        data["B_zeta_t"][4:-4],
+        B_sub_zeta_t[4:-4],
         rtol=rtol,
         atol=atol * np.mean(np.abs(data["B_zeta_t"])),
     )
     np.testing.assert_allclose(
-        data["B_zeta_tt"][2:-2],
-        B_sub_zeta_tt[2:-2],
+        data["B_zeta_tt"][4:-4],
+        B_sub_zeta_tt[4:-4],
         rtol=rtol,
         atol=atol * np.mean(np.abs(data["B_zeta_tt"])),
     )
     np.testing.assert_allclose(
-        data["|B|_t"][2:-2],
-        Bmag_t[2:-2],
+        data["|B|_t"][4:-4],
+        Bmag_t[4:-4],
         rtol=rtol,
         atol=atol * np.mean(np.abs(data["|B|_t"])),
     )
     np.testing.assert_allclose(
-        data["|B|_tt"][2:-2],
-        Bmag_tt[2:-2],
+        data["|B|_tt"][4:-4],
+        Bmag_tt[4:-4],
         rtol=rtol,
         atol=atol * np.mean(np.abs(data["|B|_tt"])),
     )
     np.testing.assert_allclose(
-        data["B_t"][2:-2, :],
-        B_t[2:-2, :],
+        data["B_t"][4:-4],
+        B_t[4:-4],
         rtol=rtol,
         atol=atol * np.nanmean(np.abs(data["B_t"])),
     )
     np.testing.assert_allclose(
-        data["B_tt"][2:-2, :],
-        B_tt[2:-2, :],
+        data["B_tt"][4:-4],
+        B_tt[4:-4],
         rtol=rtol,
         atol=atol * np.nanmean(np.abs(data["B_tt"])),
     )
 
     # partial derivatives wrt zeta
-    rtol = 1e-4
-    atol = 1e-4
-    num_zeta = 130
+    rtol = 1e-3
+    atol = 1e-3
+    num_zeta = 180
```

Comment:
Is there a reason we need to increase the tolerances, despite also increasing the finite difference resolution?

Reply:
that's fine for `rtol`, but in general could be dangerous for `atol` since some derivates might be close to zero